### ISSUE TITLE
Why are the joints regressed from posed vertices?

### ISSUE THREAD

**Issue by isarandi** [yohanshin/WHAM#67]
Thanks for the great work! I have a question about the evaluation. Currently, the joint positions are computed for both GT and prediction by applying the joint regressor to the posed mesh vertices. This won't be the same as the actual joints of SMPL given as the joint output from the SMPL model (since the standard way is to apply the regressor after applying the shape blendshapes but before posing). Is there any particular reason for not using the joint output from SMPL? Thanks!

**Comment by isarandi** [comment]
Also, why is a H36M regressor and 14 joints used for 3DPW? Why not evaluate the SMPL joints like on RICH and EMDB?

**Comment by yohanshin** [comment]
Hi @isarandi 

Thank you for your feedback! To be honest, I thought `smpl_output.joints` was identical to the joints regressed from the vertices. I borrowed the code from BEDLAM's evaluation [script](https://github.com/pixelite1201/BEDLAM/blob/ebf8bb14a43de46cc74dca4c00c13e571b325726/train/core/hmr_trainer_smpl.py#L142), where they also computed 3D joints through the regression matrix. I will review other methods' evaluation metrics and follow the convention (joints obtained from SMPL's forward pass and J_regressor are quite similar, MPJPE ~ 3mm). But thank you for the heads up!

Furthermore, we also followed the convention of the field that they compute 3DPW based on the J14 format.
